ui/package.json

```diff
@@ -190,6 +189,7 @@
     "qunit": "^2.19.1",
     "qunit-dom": "^2.0.0",
     "route-recognizer": "^0.3.4",
+    "sass": "^1.58.3",
```

Comment:
did we want the carrot here or did we want to lock it in?

Reply:
This is something that I've wanted to revisit so thanks for the reminder. I'm going to leave as is for now and I've added an item to our next sync.